docs/4.4/enterprise/ssh-rbac.md

```diff
@@ -272,11 +271,11 @@ spec:
       'environment': ['test', 'staging']
       # regular expressions are also supported, for example the equivalent
       # of the list example above can be expressed as:
-      'environment': "{% raw %}{{regexp.match("^test|staging$")}}{% endraw %}"
+      'environment': '{% raw %}{{regexp.match("^test|staging$")}}{% endraw %}'
       # or using the simpler legacy syntax:
       'environment': '^test|staging$'
       # negative regular expressions can be used to avoid strict deny rules:
-      'environment': "{% raw %}{{regexp.not_match("prod")}}{% endraw %}"
+      'environment': "{% raw %}{{regexp.not_match('prod')}}{% endraw %}"
```

Comment:
```suggestion
      'environment': '{% raw %}{{regexp.not_match("prod")}}{% endraw %}'
```

Reply:
I just checked and this format (`'` on the outside, `"` on the inside) works, but the other doesn't.